app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -211,6 +215,14 @@ public static MediaDetailFragment forMedia(int index, boolean editable, boolean
     private Media media;
     private ArrayList<String> reasonList;
 
+    /**
+     * Height stores the height of the frame layout as soon as its intialised and updates itself on
+     * configuration change.
+     * Used to adjust aspect ratio of image when length of the image is too large.
+     */
+    private int heightFrameLayuot;
+
+    private int minimumHeightOfMetadata=200;
```

Comment:
Spaces before and after `=`.
javadoc, including the unit (pixels?).
Also, this might be misunderstood as "_height found in the metadata_" so better make it `minimumHeightOfMetadataLayout` or similar (feel free to replace `Layout` with any better word).